ui/ducks/app/app.js

```diff
@@ -388,6 +405,19 @@ export function hideWhatsNewPopup() {
 export function toggleGasLoadingAnimation(value) {
   return { type: actionConstants.TOGGLE_GAS_LOADING_ANIMATION, value };
 }
+export function addTxToFailedTxesToDisplay(txId) {
```

Comment:
Could we rename these two functions and the actions as well? This isn't "failed transactions to display" anymore, it's "transactions to display on failure".

So this could be `ADD_TRANSACTION_TO_DISPLAY_ON_FAILURE` and `addTransactionToDisplayOnFailure` instead for example.